components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/gatewayartifactsynchronizer/ArtifactRetriever.java

```diff
@@ -26,6 +28,16 @@
     String retrieveArtifact(String APIId, String gatewayLabel, String gatewayInstruction)
             throws ArtifactSynchronizerException;
 
+    /**
+     * This method is used to retrieve data from the storage
+     *
+     * @param gatewayLabel       - Label subscribed by the gateway
```

Comment:
We don't use '-' (dash) as a separator

Reply:
https://github.com/wso2/carbon-apimgt/pull/9546/commits/ad587f2f16bdef1c6b05054c42b780dda8e218b9